src/lib/y2storage/disk.rb

```diff
@@ -0,0 +1,101 @@
+require "y2storage/storage_class_wrapper"
+require "y2storage/partitionable"
+require "y2storage/free_disk_space"
+
+module Y2Storage
+  # A physical disk device
+  #
+  # This is a wrapper for Storage::Disk
+  class Disk < Partitionable
+    include StorageClassWrapper
+    wrap_class Storage::Disk
+
+    storage_forward :rotational
+    alias_method :rotational?, :rotational
```

Comment:
well, maybe alias is not good name, maybe ruby_name is better. What I would like to see is, no `rotational` method and only `rotational?` one